src/main/java/com/glodblock/github/coremod/transform/DualityInterfaceTransformer.java

```diff
@@ -39,9 +40,42 @@ public MethodVisitor visitMethod(int access, String name, String desc, String si
     }
 
     private static class TransformInvAdaptorCalls extends MethodVisitor {
+        private final String name;
+        private Label label;
+        private boolean correctLabel;
 
-        TransformInvAdaptorCalls(int api, MethodVisitor mv) {
+        private TransformInvAdaptorCalls(int api, MethodVisitor mv, String name) {
             super(api, mv);
+            this.name = name;
+        }
+
+        @Override
+        public void visitFieldInsn(int opcode, String owner, String name, String desc) {
+            super.visitFieldInsn(opcode, owner, name, desc);
+            if (this.name.equals("pushItemsOut") && opcode == Opcodes.PUTFIELD && owner.equals("net/minecraft/item/ItemStack") && name.equals("stackSize")) {
```

Comment:
what here is doing, too much asm code to read

Reply:
make sure correct code line.